src/main/resources/messages.properties

```diff
@@ -5,6 +5,7 @@
 invalid.not-positive.id=ID value has to be positive
 invalid.not-empty=${validatedValue} has to be not empty
 invalid.not-null=${validatedValue} has to be not null
+invalid.not-null-phone=phone number should not be null
```

Comment:
You should keep ${validatedValue} for that message as well. That is our agreement with FE guys